test/common_quantization.py

```diff
@@ -116,6 +116,13 @@ def checkQuantizedLinear(self, mod):
     def checkLinear(self, mod):
         self.assertEqual(type(mod), torch.nn.Linear)
 
+    def checkScriptable(self, mod, ref_inputs=None, ref_outputs=None):
+        assert not ((ref_inputs is None) ^ (ref_outputs is None)), 'Both or neither' \
+            'of ref_inputs and ref_outputs must be specified'
+        scripted = torch.jit.script(mod)
+
+        if ref_inputs is not None:
```

Comment:
Can we remove the if here and mandate that ref_input is provided?

Reply:
No, because I use this in `test_quantized.py` in which the modules are not actually run